test/helpers/patients_helper_test.rb

```diff
@@ -137,4 +181,67 @@ class PatientsHelperTest < ActionView::TestCase
                            voicemail_options('ID as coworker')
     end
   end
+
+  describe 'language_options' do
+    before { create_language_config }
+
+    it 'should return default languages' do
+      expected_lang_options = [
+        ['English', nil],
+        'Spanish',
+        'French',
+        'Korean'
+      ]
+      assert_same_elements expected_lang_options, language_options()
+    end
+
+    it 'should append extra options' do
+      expected_lang_options = [
+        ['English', nil],
+        'Spanish',
+        'French',
+        'Korean',
+        'Esperanto'
+      ]
+      assert_same_elements expected_lang_options, language_options('Esperanto')
+    end
+  end
+
+  describe 'referred_by_options' do
+    before { create_referred_by_config }
+
+    # base options come from patients_helper
+    # Metal band is added by create_referred_by_config
+    expected_referrals_base = [
+      nil,
+      ["Clinic", "Clinic"],
+      ["Crime victim advocacy center", "Crime victim advocacy center"],
+      ["DCAF website or social media", "DCAF website or social media"],
+      ["Domestic violence crisis/intervention org", "Domestic violence crisis/intervention org"],
+      ["Family member", "Family member"],
+      ["Friend", "Friend"],
+      ["Google/Web search", "Google/Web search"],
+      ["Homeless shelter", "Homeless shelter"],
+      ["Legal clinic", "Legal clinic"],
+      ["NAF", "NAF"],
+      ["NNAF", "NNAF"],
+      ["Other abortion fund", "Other abortion fund"],
+      ["Previous patient", "Previous patient"],
+      ["School", "School"],
+      ["Sexual assault crisis org", "Sexual assault crisis org"],
+      ["Youth outreach", "Youth outreach"],
+      "Metal band"
+    ]
+
+    it 'should return default referral options' do
+      assert_same_elements expected_referrals_base, referred_by_options()
+    end
+
+    it 'should append extra options' do
+      expected_referrals = expected_referrals_base + ['Social Worker']
+
+      assert_same_elements expected_referrals, referred_by_options('Social Worker')
```

Comment:
I didn't know this was an assertion in the minitest toolbox! neat

Reply:
I picked it up from somewhere in the test cases! But it is super helpful.